app/src/main/res/values/strings.xml

```diff
@@ -36,40 +36,42 @@
 	<string name="share_s">Share %s</string>
 	<string name="search">Search</string>
 	<string name="search_manga">Search manga</string>
-	<string name="manga_downloading_">Manga downloading…</string>
+	<string name="manga_downloading_">Downloading…</string>
 	<string name="processing_">Processing…</string>
-	<string name="download_complete">Download complete</string>
+	<string name="download_complete">Downloaded</string>
 	<string name="downloads">Downloads</string>
-	<string name="by_name">By name</string>
+	<string name="by_name">Name</string>
 	<string name="popular">Popular</string>
 	<string name="updated">Updated</string>
 	<string name="newest">Newest</string>
-	<string name="by_rating">By rating</string>
+	<string name="by_rating">Rating</string>
 	<string name="all">All</string>
-	<string name="sort_order">Sort order</string>
+	<string name="sort_order">Sorting order</string>
 	<string name="genre">Genre</string>
 	<string name="filter">Filter</string>
 	<string name="theme">Theme</string>
 	<string name="light">Light</string>
 	<string name="dark">Dark</string>
+	<string name="black_dark_theme">Black</string>
```

Comment:
Black what?

Reply:
This string used as a preference title, so just "Black" is not clear to understand

Reply:
It is black, and that is what I've seen other apps use.
Some have a less than 100% black option called "dark".